cwl/src/main/scala/wdl4s/cwl/CwlExpressionCommandPart.scala

```diff
@@ -0,0 +1,15 @@
+package wdl4s.cwl
+
+import wdl4s.wdl.values.WdlValue
+import wdl4s.wom.CommandPart
+import wdl4s.wom.expression.IoFunctionSet
+
+
+case class CwlExpressionCommandPart(expr: String) extends CommandPart {
+  override def instantiate(inputsMap: Map[String, WdlValue],
+                            functions: IoFunctionSet,
+                            valueMapper: (WdlValue) => WdlValue): String = {
+    CwlWomExpression(expr).evaluateValue(inputsMap, functions).map(_.valueString).getOrElse(???)
```

Comment:
ToL (esp @kshakir) - maybe we can make this return a `Future` (or some other wrapper)...?

Reply:
Yeah, at least an `ErrorOr` but if evaluateValue returned a `Future` and hence instantiate too  that would be even better IMO